.github/workflows/canary.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Install dependencies
         run: npm ci
         env:
-          PUPPETEER_SKIP_DOWNLOAD: true
```

Comment:
let's not change this one, PUPPETEER_SKIP_DOWNLOAD means skipping all download step during installation

Reply:
Done.